content/roadmap/intro.md

```diff
@@ -1 +1,9 @@
-This roadmap shows what the Design System team is currently planning to work on over the coming year. This roadmap is open to change and we welcome the [community input](https://community.digital.gov.au/t/roadmap-discussion/68). For more detail, including the teams current and upcoming sprint goals, please see the [public project board](https://github.com/orgs/govau/projects/7)
+---
+layout: datemetadata/metadata-datetime
+datePublished: 28 Feb 2019
+dateModified: 29 May 2019
+timePublished: "2019-02-28"
+timeModified: "2019-05-29"
+---
```

Comment:
I would recommend using an object array here. Something like this:

![image](https://user-images.githubusercontent.com/20184809/58677464-5f18a480-839f-11e9-84fb-b492e99413b5.png)

so maybe someting like:
```
descriptionlList:
  - dt: data
    dd: data
```
etc.

Reply:
Userful reference :

https://cuttlebelle.com/cheatsheet/